README.md

```diff
@@ -46,12 +45,11 @@ Executing `make` on the command line will give you a list of possible commands t
 In order to create a new stack you can run these make commands in order:
 
 ```shell
-# ensure that you have set up and sourced your environment variables in `environment.sh`
+# ensure that you have set up and sourced your environment variables using `source environment.sh`
 
 make create-stack   # Create the terraform stack env vars
 make create-bucket  # Create the terraform state bucket
 make init           # Initialise terraform
-make plan           # Plan all terraform
```

Comment:
We decided to remove it as the apply, will also run the plan and in the initial creating of a new stack there will be errors from the plan as resources are not available. So for a fresh stack we will need to apply each project so that the plans pass.

It is left in so that developers can see the impact of any changes without applying them in an existing stack.